extensions.bzl

```diff
@@ -3,6 +3,37 @@
 load("@bazel_tools//tools/build_defs/repo:cache.bzl", "get_default_canonical_id")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "get_auth")
 
+def _get_current_platform(ctx):
+    """
+    Get the current platform.
+
+    It uses the context to get the current platform and architecture and returns the platform name.
+    The string returned is one of the following:
+
+    - windows
+    - mac
+    - mac-arm
+    - linux
+    - linux-arm
+
+    Args:
+        ctx: a [repository context](https://bazel.build/rules/lib/builtins/repository_ctx) object
+            or a [module context](https://bazel.build/rules/lib/builtins/module_ctx) object
+            containing the repository's attributes
+    """
+
+    if ctx.os.name.startswith("windows"):
+        return "windows"
+    if ctx.os.name.startswith("mac") and (ctx.os.arch == "amd64" or ctx.os.arch == "i686"):
```

Comment:
Some of our CI/CQ tryjobs are failing here because a specific `arch` string isn't mentioned here. Will follow up tomorrow AM with the suggested updates